dz_mongodb/db_utils.py

```diff
@@ -227,4 +210,9 @@ def produce_collection_schema(collection: Collection) -> Dict:
 
             db_schema['schema']['properties'][key] = {"type": val_type}
 
+    # exposing all fields to be incremental
```

Comment:
did we check existing source? what happens when we try to modify it?

Reply:
basically if the key that was already there shouldn't change

Reply:
this is only being used to select incremental cursor and in existing source no one connected incremental.